src/react-components/src/components/02-molecules/Menus/Pagination/Pagination.tsx

```diff
@@ -57,7 +56,7 @@ export default function Pagination(props: PaginationProps) {
   return (
     <nav aria-label="Pagination" className={bem(pagination__base_class)}>
       <Button callback={buttonPrevOpts.callback} {...buttonPrevOpts}>{buttonPrevOpts.content}</Button>
-      <Dropdown labelPosition="left" {...dropdownProps}></Dropdown>
+      <div className={bem("dropdown-container", [], pagination__base_class)}><Dropdown labelPosition="left" {...dropdownProps}></Dropdown></div>
```

Comment:
Could `Dropdown` allow us to pass these values to it instead of adding them to a div just outside of it?

Reply:
haha I wrote out a whole thing about how custom classes are really tricky here but it turns out https://github.com/NYPL/nypl-design-system/blob/development/src/react-components/src/components/02-molecules/Dropdown/Dropdown.tsx#L60 never has a baseClass passed into it and I think that's probably the best place to put these styles.

Reply:
Yes, same as button. I'd modify `Dropdown` to take a `blockName` and then do the same as button here